### PR TITLE
Angle switch

### PR DESCRIPTION
**Pull request by PrathamRawat**

### CODE REVIEW

File: src/org/usfirst/frc/team694/robot/Robot.java [hunk 1]
```diff
@@ -56,16 +46,27 @@ public void robotInit() {
         grabber = new Grabber();
         lift = new Lift();
         oi = new OI();
-        
-<<<<<<< HEAD
-       
-=======
+        currentQuad = getRobotQuadrant();
```
Comment (TacoTechnica) [inline src/org/usfirst/frc/team694/robot/Robot.java]: Don't do this. Here we have to assume that this variable is set. Just use getRobotQuadrant() in each command.

**Review by TacoTechnica — DISMISSED** [review]
Remove the public currentQuad variable from Robot

File: .classpath [hunk 2]
```diff
@@ -10,5 +10,6 @@
 	<classpathentry kind="var" path="USERLIBS_DIR/CTRE_Phoenix-sources.jar"/>
 	<classpathentry kind="var" path="USERLIBS_DIR/CTRE_Phoenix.jar"/>
 	<classpathentry kind="var" path="USERLIBS_DIR/navx_frc.jar"/>
+	<classpathentry kind="var" path="USERLIBS_DIR/CTRLib.jar"/>
```
Comment (nicoleshin) [inline .classpath]: don't push your class path

**Review by nicoleshin — APPROVED** [review]
+1

**Pull request merged**